docs/source/user_guide/01a_interface.rst

```diff
@@ -0,0 +1,29 @@
+
+nbgrader interface highlights
```

Comment:
Probably same thing here -- I'm fine having these things be in the docs (that's actually probably a better place for them) but then I think we should remove the redundant information in the README and link to the docs.